src/UI/Components/Diagram/actions.ts

```diff
@@ -258,6 +271,7 @@ export function appendEmbeddedEntity(
   holderType: string,
   instanceToConnectRelation?: ServiceEntityBlock,
   presentedAttr?: "candidate" | "active",
+  isBlockedFromEditing?: boolean,
```

Comment:
isEditable?

Reply:
I prefer to go by triggering only when some instance is blocked from editing, so by default newly created instances doesn't have to have extra parameter in that approach

Reply:
isEditable is for me just meaning the same, it's what's used on other places too. I don't see how changing the name would make it any different when initializing the instances, or how that would mean an additional parameter.

Reply:
For me, semantically `IsEditable === !isBlockedFromEditing`, and to keep in line naming and implementation then I would have to add that every possible shape is editable by default, which is redundant. I can change naming, but then in 6 months it could be confusing that something is set to true but its blocked from editing :)